custom_components/unifiprotect/camera.py

```diff
@@ -160,6 +160,11 @@ def _async_updated_event(self):
         )
         self.async_write_ha_state()
 
+    @property
+    def supported_features(self):
```

Comment:
😢 

Camera doesn't support `_attr` yet
https://github.com/home-assistant/core/blob/dev/homeassistant/components/camera/__init__.py#L393

Reply:
Oh, is that a newer things some of the other base classes supports?

Reply:
https://developers.home-assistant.io/docs/core/entity/#entity-class-or-instance-attributes